src/atoms/Layout/layout.module.scss

```diff
@@ -45,11 +55,16 @@ $border: 1px solid color('neutral-6');
     [class*='col-'] {
       border-left: $border;
       border-left: $border;
-      padding: ru(.75) ru(.5);
+      padding: 0 ru(.5) ru(1.25);
```

Comment:
@patrickkim we still want the padding around the children that are in the layout - just the left padding on the first child should be 0, and then on small screen sizes there should be ru(1) margin/padding bottom - looks like at the moment it's ru(0.5)